Classes/GameMapScene.cpp

```diff
@@ -134,14 +122,14 @@ void GameMap::setupButtonExit()
 void GameMap::setupLabels()
 {
 	//score
-	scoreLabel = Label::createWithTTF("Estrellas: 0%", "fonts/Akashi.ttf", 18);
+	scoreLabel = Label::createWithTTF("Stars: 0", "fonts/Andika-Regular.ttf", 24);
```

Comment:
@protito How do we implement i18n, so that "Stars" is used on English tablets and "Nyota" is used on Swahili tablets? Is it possible to separate such texts into separate i18n properties files?